### PR TITLE
fix: use the correct preset for windows repo list & add additional field to aksengine_helpers.go

### PR DESCRIPTION
**Pull request by chewong**
Signed-off-by: Ernest Wong <chuwon@microsoft.com>

- <s>fixes failure in https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-aks-engine-azure-master-windows-containerd/1393315723899572224</s>
- use the correct windows repo list (use preset-windows-repo-list when <1.20, otherwise use preset-windows-repo-list-master)
- bump --aksengine-orchestratorRelease from 1.21 to 1.22 for master jobs

### CODE REVIEW

File: kubetest/aksengine_helpers.go [hunk 1]
```diff
@@ -140,41 +140,51 @@ type KubernetesAddon struct {
...
+	WindowsContainerdURL             string                `json:"windowsContainerdURL,omitempty"`
+	WindowsSdnPluginURL              string                `json:"windowsSdnPluginURL,omitempty"`
+	Components                       []KubernetesComponent `json:"components,omitempty"`
+}
+
+// KubernetesComponent defines a component w/ configuration to include with the cluster deployment
+type KubernetesComponent struct {
+	Name       string                    `json:"name,omitempty"`
+	Enabled    *bool                     `json:"enabled,omitempty"`
+	Containers []KubernetesContainerSpec `json:"containers,omitempty"`
```
Comment (jsturtevant) [inline kubetest/aksengine_helpers.go:185]: does this also need to be specified?

Comment (chewong) [reply kubetest/aksengine_helpers.go:185]: Yes, we need to parse the following new JSON fields I introduced in https://github.com/kubernetes-sigs/windows-testing/pull/267
```json
        "components": [
          {
            "name": "kube-addon-manager",
            "enabled": true,
            "containers": [
              {
                "name": "kube-addon-manager",
                "image": "mcr.microsoft.com/oss/kubernetes/kube-addon-manager:v9.1.5"
              }
            ]
          }
        ]
```

**Comment by chewong** [comment]
/assign @feiskyer 
for kubetest

File: kubetest/aksengine_helpers.go (hunk 1 above)
Comment (chewong) [reply kubetest/aksengine_helpers.go]: Addressing kubetest-related change in a separate PR

**Pull request merged**